BUILD.gn

```diff
@@ -32,6 +33,18 @@ if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
         (host_cpu == "x64" || host_cpu == "arm64")
   }
 
+  pw_python_package("happy") {
```

Comment:
nit: Move to `third_party/happy/BUILD.gn`

The examples don't load this build file (try saying `${chip_root}:happy` from one of those - it'll hit the assert at the top of this file).

This is basically for organizational purposes, but it serves a useful purpose of preventing examples from bringing in other examples, and it's enforced.

Reply:
that's a third party repo, we may not want to add dependency on PW to that

Reply:
oh, outside of the repo subdir, okay

Reply:
This was not added for the example btw, this was needed because of the Pigweed rollup